### PR TITLE
Merge files from origin project

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Lukianov Artyom <alukiano@redhat.com>

### CODE REVIEW

**Comment by davidvossel** [comment]
It's fine to move in your repo as one big commit if you want.  You could also move over all the commits from the original repo if you wanted to keep your commit history.  It doesn't matter to me. I just wanted to point out you could keep your commit history if it is important to you.

**Comment by cynepco3hahue** [comment]
@davidvossel Thanks for proposal :smile:, but it does not really matter for me, all my future patches will be for this repository.

**Comment by fabiand** [comment]
@cynepco3hahue why do you want to import the complete openshift deployment parts?

Can't they be referenced?

**Comment by cynepco3hahue** [comment]
@fabiand What do you mean by `complete openshift deployment parts`? I run some prerequisite package installation and docker-storage-setup, after it I just include openshift-ansible playbook `- include: "{{ openshift_ansible_dir }}/playbooks/byo/config.yml"`

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# kubevirt-ansible
\ No newline at end of file
+# KubeVirt-ansible
+The purpose of this project to create Kubernetes or OpenShift cluster and deploy KubeVirt environment on it.
+Tested on CentOS Linux release 7.3.1611 (Core).
+
+### Requirments
```
Comment (lukas-bednar) [inline README.md]: s/Requirments/Requirements/

File: common/roles/kubevirt/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (fabiand) [inline common/roles/kubevirt/tasks/main.yml]: This task should somehow be called something related to `development`.
After all installing libvirt on the host and cloning the source repository is the development flow.

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+Install depending roles, and export `ANSIBLE_ROLES_PATH`
+```
+$ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
+$ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
+```
+For OpenShift deployment clone [**OpenShift Ansible project**](https://github.com/openshift/openshift-ansible)
+```
+$ git clone https://github.com/openshift/openshift-ansible
+```
+
```
Comment (fabiand) [inline README.md]: Please add an example of how I can deploy kubevirt to an existing cluster

Comment (petrkotas) [reply README.md]: @fabiand Would you be OK if we continue and do polishing after we merge?

Comment (fabiand) [reply README.md]: Sure

File: common/roles/kubevirt/tasks/main.yml (hunk 2 above)
Comment (fabiand) [reply common/roles/kubevirt/tasks/main.yml]: This is also non-blocking for the merge.

File: README.md (hunk 3 above)
Comment (fabiand) [reply README.md]: I've filed #5

Comment (petrkotas) [reply README.md]: Thank you, lets do this :)

**Pull request merged**